### PR TITLE
Add DNS leak tests for desktop and Android; fixes issue #235

### PR DESCRIPTION
**Pull request by stephendonner**
I tried for parity, but Android doesn't have the scheme handler that we do on desktop, and perfect is the enemy of good.

I'll squash commits before merging, once approved.

r? @kjozwiak @diracdeltas @srirambv?

### CODE REVIEW

File: wikitemplate.md [hunk 1]
```diff
@@ -67,6 +67,7 @@
 - [ ] Visit https://www.blizzard.com and then turn on script blocking, page should not load.
 - [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/9/ when 3rd party cookies are blocked and not blank when 3rd party cookies are unblocked
 - [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run)
+- [ ] In `brave://settings/security`, choose a DNS provider from the providers listed under Use secure DNS, load https://browserleaks.com/dns, and verify your ISP's DNS resolvers aren't detected and shown; only your chosen DoH provider should appear.
```
Comment (diracdeltas) [inline wikitemplate.md:70]: :+1: this should also have an item for `Open a Tor tab, load https://browserleaks.com/dns, and verify your ISP's DNS resolvers aren't detected and shown`

Comment (stephendonner) [reply wikitemplate.md:70]: Just added: https://github.com/brave/qa-resources/pull/236/commits/07602478a344efd02dc0d05295f89562eaef4ea4#diff-8e23795bde2c11770f8ae9394839337a39d31831496f063cc8d2e953569767dbR71

**Review by diracdeltas — APPROVED** [review]

**Review by kjozwiak — CHANGES_REQUESTED** [review]
@stephendonner looking good 👍 Just a small nit. Lets follow the format of adding links into code formatting.. Example:

`https://browserleaks.com/dns` vs https://browserleaks.com/dns

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**